resize_dataset/dataset/coco.py

```diff
@@ -122,6 +122,8 @@ def __init__(self, images_path, annotations_path, cfg):
         self.images_output_folder = self.cfg.images_output_path
         self.labels_output_path = self.cfg.labels_output_path
         self.output_annotations = self.annotations.dataset
+        self.output_annotations["annotations"] = []
```

Comment:
With this initialization I guess we only want to get the info and categories from the original json. I think this is correct.